src/server/search/search_family.cc

```diff
@@ -1013,19 +1014,28 @@ void SearchReply(const SearchParams& params,
 
   const bool reply_with_ids_only = params.IdsOnly();
   auto* rb = static_cast<RedisReplyBuilder*>(builder);
-  RedisReplyBuilder::ArrayScope scope{rb, reply_with_ids_only ? (limit + 1) : (limit * 2 + 1)};
+  const size_t items_per_field = (reply_with_ids_only ? 1 : 2) + params.with_sortkeys;
+  RedisReplyBuilder::ArrayScope scope{rb, limit * items_per_field + 1};
+
+  Overloaded sortable_value_sender{
+      [rb](monostate) { rb->SendNull(); },
+      [rb](double d) { rb->SendBulkString(absl::StrCat("#", d)); },
+      [rb](const string& s) { rb->SendBulkString("$" + s); },
+  };
```

Comment:
Thats the format they use? Because they use resp2 for search and there is no type distinction?